addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2689,19 +2689,33 @@ export class OdooEditor extends EventTarget {
      */
     _positionTableUi(element) {
         const tableUiContainerRect = this._tableUiContainer.getBoundingClientRect();
+        const isRtl = this.options.direction === 'rtl';
         const isRow = element.nodeName === 'TR';
         const ui = isRow ? this._rowUi : this._columnUi;
         const elementRect = element.getBoundingClientRect();
         const wrappedUi = ui.firstElementChild;
-        const table = getInSelection(this.document, 'table');
+        const table = closestElement(element, 'table');
+        const tableRect = table && table.getBoundingClientRect();
         const resetTableSize = ui.querySelector('.o_reset_table_size');
-        if (table && !table.hasAttribute('style')) {
+        if (!table.hasAttribute('style')) {
```

Comment:
Why do you test that `table` exists when defining `tableRect` but remove the check here?

Reply:
That's a mistake, I added back the check in the if

Reply:
You did? Did you forget to push?

Reply:
Indeed 👍